usagov/usagov-2021
# usagov-1330-analytics-descriptive-link: add descriptive text to link

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1330

## Description

Link text of ' download the data here '  has no meaning out of context. Add textual meaning to the link.
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Use VS code to view all the index.html files in the website-analytics folder.
- You can use Go Live to view each index.html page to verify the text at the bottom right now says ' download the website analytics data here ' instead of ' download the data here '.
- You can also so a search and find for the text ' download the website analytics data here ' in each index.html file.
- 
<img width="695" alt="Screenshot 2024-06-05 at 10 18 28 AM" src="https://github.com/usagov/usagov-2021/assets/5551880/e19cd5ae-90f4-4b56-a6c0-c7fe4ee261bf">


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
